features/step_definitions/work_steps.rb

```diff
@@ -249,6 +249,51 @@
   click_button("Preview")
 end
 
+When /^I set the fandom to "([^\"]*)"$/ do |fandom|
+  fill_in("Fandoms", with: fandom)
+end
+
+When /^I select "([^\"]*)" for editing$/ do |title|
+  id = Work.find_by_title(title).id
+  check("work_ids_#{id}")
+end
+
+When /^I edit the multiple works "([^\"]*)" and "([^\"]*)"/ do |title1, title2|
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.